src/renderer/components/database-diagram.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { Component, PropTypes } from 'react';
+import joint from 'jointjs';
+
+require('./jointjs-diagram-table.js');
+require('./jointjs-diagram-table-cell.js');
+require('jointjs/dist/joint.min.css');
```

Comment:
Hmm, so you wanted to suggest moving JointJS external styles to `app.jsx` or just above requiring jointjs-diagram elements?